### DISCUSSION TITLE
Flow cell for industrial probes

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#512] [General]
Industrial probes are en route for ORP and PH.  I've got the Hayward flow cell everyone else does, but hesitant to modify it.  I have a 
Clear Solid Acrylic Display Block - 6" x 4" x 2.36" on its way for a date with my Bridgeport.  I also picked up another flow cell flow switch.

I'll sketch up what I'm thinking for machining it, but other than the flow switch and two industrial probes, can anyone think of what else might be needed?

**Comment by johnny2678** [comment]
> I've got the Hayward flow cell everyone else does, but hesitant to modify it.

This is 100% the reason I am not running the industrial probes.  yea, @rstrouse makes it sound easy in the wiki, but I'm not so confident.

I don't have a bridgeport, would have to buy a 59/64" bit and hand drill, and I've never used an NTP tap (though I do own the 3/4 ntp bit/hand drill). 😂

Would probably spring for the combined orp/ph/temp industrial probe if I felt like I could pull this off.

**Comment by rstrouse** [comment]
If you do end up modifying your existing flow cell to 3/4npt you can simply get 3/4in probe inserts if you want to use consumer or lab grade probes.  With a milling machine I suspect you can be very precise.  Interested to see your milled flow cell though.

A test port might be nice.  Not necessarily for testing but if you want to drain the water out of the cell it is handy.

I have done this 3 times and never screwed it up.  Once with a hand drill and twice with a drill press.  However, I have a neighbor who brought me a mangled mess because his drill press had so much runout that the bit looked like a noodle as it was spinning.

The bummer is that if you did screw it up, you used to be out 40 bucks for a new cell.  Now I only see them with a flow switch and valves for 150 bucks.

**Comment by orrious** [comment]
How much water do we want in the flow cell?  I.e the Hayward Flow Cell holds quite a bit of water.  The acrylic one that I'm attempting to fabricate will not hold much, but it will keep the probes wet when the pump is off.

**Reply by tagyoureit** [reply]
It doesn't have to be much.  You want a constant flow over the probes so they are always getting a fresh supply.  The biggest concern would be air bubbles.  The flow cells do a nice job of 'tornadoing' the water such that the air bubbles (if any) are pulled away from the probe tips.
